base/replutil.jl

```diff
@@ -666,3 +666,14 @@ function process_backtrace(process_func::Function, t::Vector, limit::Int=typemax
         process_func(last_frame, n)
     end
 end
+
+"""
+Determines whether a method is the default method which is provided to all types from sysimg.jl.
+Such a method is usually undesirable to be displayed to the user in the REPL.
+"""
+function is_default_method(m::Method)
+    if m.module == Base && m.file == Symbol("sysimg.jl") && m.sig == Tuple{Type{T},Any} where T
+        return true
+    end
+    return false
```

Comment:
This can be simplified to `return m.module == Base && m.file == Symbol("sysimg.jl") && m.sig == Tuple{Type{T},Any} where T`